app/Console/Commands/SendExpirationAlerts.php

```diff
@@ -44,79 +47,25 @@ public function __construct()
     public function fire()
     {
 
+        $threshold = Setting::getSettings()->alert_interval;
         // Expiring Assets
-        $expiring_assets = Asset::getExpiringWarrantee(Setting::getSettings()->alert_interval);
-        $this->info(count($expiring_assets).' expiring assets');
-
-        $asset_data['count'] =  count($expiring_assets);
-        $asset_data['email_content'] ='';
-        $now = date("Y-m-d");
-
-
-        foreach ($expiring_assets as $asset) {
-
-            $expires = $asset->present()->warrantee_expires();
-            $difference =  round(abs(strtotime($expires) - strtotime($now))/86400);
-
-            if ($difference > 30) {
-                $asset_data['email_content'] .= '<tr style="background-color: #fcffa3;">';
-            } else {
-                $asset_data['email_content'] .= '<tr style="background-color:#d9534f;">';
-            }
-            $asset_data['email_content'] .= '<td><a href="'.config('app.url').'/hardware/'.e($asset->id).'/view">';
-            $asset_data['email_content'] .= $asset->present()->name().'</a></td><td>'.e($asset->asset_tag).'</td>';
-            $asset_data['email_content'] .= '<td>'.e($asset->present()->warrantee_expires()).'</td>';
-            $asset_data['email_content'] .= '<td>'.$difference.' '.trans('mail.days').'</td>';
-            $asset_data['email_content'] .= '<td>'.($asset->supplier ? e($asset->supplier->name) : '').'</td>';
-            $asset_data['email_content'] .= '<td>'.($asset->assignedTo ? e($asset->assignedTo->present()->name()) : '').'</td>';
-            $asset_data['email_content'] .= '</tr>';
-        }
+        $assets = Asset::getExpiringWarrantee(Setting::getSettings()->alert_interval);
+        $this->info($assets->count().' assets expiring within '.Setting::getSettings()->alert_interval.' days');
```

Comment:
These (and below) could use $threshold, right?

Reply:
Yeah, I'm adding new mail strings. The way we broke it up before ("There is|There are" as one string and the rest of the sentence as a separate string is dumb and doesn't make sense for non-english grammar.